pkgs/shells/fzf-git-sh/default.nix

```
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fzf-git-sh";
-  version = "unstable-2022-09-30";
+  version = "unstable-2024-02-17";
```

```suggestion
  version = "0-unstable-2024-02-17";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming